.github/workflows/website.yml

```diff
@@ -0,0 +1,156 @@
+name: Website
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  docs-home:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+
+      - run: corepack enable && yarn install --immutable
+
+      - run: yarn build
+        working-directory: website
+
+      - run: yarn typedoc
+
+      - run: |
+          mkdir -p website/build/docs
+          mv __docs__/typescript website/build/docs/typescript
```

Comment:
What builds this thing?

Reply:
`yarn typedoc` immediate above

Reply:
removed this line entirely by having typedoc output directly into the correct directory